torch/_inductor/runtime/triton_helpers.py

```diff
@@ -27,7 +27,10 @@ def set_driver_to_gpu():
     driver = triton.runtime.driver
```

Comment:
It is not easy to get `LazyProxy` type from `triton.runtime` module, since in `triton.runtime.__init__` a driver module is replaced with a driver object. https://github.com/triton-lang/triton/blob/2b2a872459648a7418d1c7e4cfa9aa8cb39e71e5/python/triton/runtime/__init__.py#L3